packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -149,7 +149,11 @@ function Thread( {
 						<VStack className="editor-collab-sidebar-panel__show-more-reply">
 							{ sprintf(
 								// translators: %s: number of replies.
-								_x( '%s more replies', 'Show replies button' ),
+								_n(
+									'%s more reply',
+									'%s more replies',
+									thread?.reply?.length
```

Comment:
Thanks for the fix. I can't seem to test this. It could be something to do with my local set up. I enabled the experiment 🤔 

Anyway, my only comment would be to cache `thread?.reply?.length` in a const above. E.g.m 

`const threadReplyLength = thread?.reply?.length;`